erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md

```diff
@@ -0,0 +1,7 @@
+<!-- add-breadcrumbs -->
+# Leave Ledger Entry
+
+A unified ledger for all leaves related transactions for an employee. This maintains all the leaves related transactions, this includes the leave allocations, leave applications and leave encashments related to your employees.
+
+<img alt="Leave Ledger Entry" class="screenshot" src="{{docs_base_url}}/assets/img/human-resources/leave-ledger-entry.png">
```

Comment:
image has a black bar at the top, crop it

Reply:
removed